Main.py

```diff
@@ -119,6 +119,14 @@
     if message.author == bot.user:
         return
 
+    if 'open.spotify' in message.content.lower():
+        new_message = message.content.translate({ord('`'): None})
+        author = message.author
+
+        await message.delete()
+        await message.channel.send(
+            'Spotify link from %s : `%s`' % (author.name, new_message))
```

Comment:
maybe can you use `"whatever".format()` strings instead for consistency with other code?

Reply:
```python
"Spotify link from {} : `%s`".format(author.name, new_message)
```